machine-learning/04-openpbta_plot_LV.Rmd

```diff
@@ -0,0 +1,379 @@
+```
+
+Add a built-in `ggplot2` theme.
+
+```{r theme_bw, live = TRUE}
+# Use @jaclyn-taroni's favorite theme :)
+ggplot(lv_plot_df,
+       aes(x = molecular_subtype, 
+           y = LV_estimate, 
+           group = molecular_subtype, 
+           colour = molecular_subtype)) +
+  geom_boxplot(outlier.shape = NA) +
+  geom_jitter(width = 0.2, alpha = 0.5) +
+  theme_bw()
+```
+
+Use the same color palette we used in the UMAP plot earlier in the module.
+
+```{r okabe_ito, live = TRUE}
+# Add colorblindr::scale_color_OkabeIto()
+ggplot(lv_plot_df,
+       aes(x = molecular_subtype, 
+           y = LV_estimate, 
+           group = molecular_subtype, 
+           colour = molecular_subtype)) +
+  geom_boxplot(outlier.shape = NA) +
+  geom_jitter(width = 0.2, alpha = 0.5) +
+  theme_bw() +
+  colorblindr::scale_color_OkabeIto()
+```
+
+Add a title to the plot.
+
+```{r add_title, live = TRUE}
+# Use labs() to add a title
+ggplot(lv_plot_df,
+       aes(x = molecular_subtype, 
+           y = LV_estimate, 
+           group = molecular_subtype, 
+           colour = molecular_subtype)) +
+  geom_boxplot(outlier.shape = NA) +
+  geom_jitter(width = 0.2, alpha = 0.5) +
+  theme_bw() +
+  colorblindr::scale_color_OkabeIto() +
+  labs(title = lv_to_plot)
+```
+
+Center the title and make it bigger and in bold.
+
+```{r center_title, live = TRUE}
+# Use theme() to improve the way the title looks
+ggplot(lv_plot_df,
+       aes(x = molecular_subtype, 
+           y = LV_estimate, 
+           group = molecular_subtype, 
+           colour = molecular_subtype)) +
+  geom_boxplot(outlier.shape = NA) +
+  geom_jitter(width = 0.2, alpha = 0.5) +
+  theme_bw() +
+  colorblindr::scale_color_OkabeIto() +
+  labs(title = lv_to_plot) +
+  theme(plot.title = element_text(size = 15,
+                                  face = "bold",
+                                  hjust = 0.5))
+```
+
+Use the next chunks to further customize your plot.
+We might suggest starting with the x- and y-axis labels.
+
+```{r labs_help}
+?labs
+```
+
+```{r adventure_time}
+
+```
+
+### Adding p-values to a plot
+
+R is great for analyzing genomic data and for publication ready visualizations, but as language for statistical computing, it's also great for statistical analysis!
+
+Let's start by performing a pairwise Wilcoxon rank sum test. 
+This is also sometimes called a Mann-Whitney U test.
+It is a non-parametric test of the null hypothesis that two groups are from the same population or that the medians of the two groups are identical. 
+_Pairwise_ means we will test all pairs of groups.
+
+We can put that information onto the plot with the `ggsignif` package.
+
+```{r pairwise_wilcox}
+pairwise.wilcox.test(x = lv_plot_df$LV_estimate,
+                     g = lv_plot_df$molecular_subtype,
+                     # Because we're performing multiple tests, we should
+                     # adjust the p-values
+                     p.adjust.method = "bonferroni")
+```
```

Comment:
@jashapiro what do you think about this wording?

Reply:
👍